pkg/internal/apis/config/encoding/testdata/v1alpha3/valid-many-fields.yaml

```diff
@@ -0,0 +1,17 @@
+kind: Cluster
+apiVersion: kind.sigs.k8s.io/v1alpha3
+networking:
+  ipFamily: ipv6
```

Comment:
doesn't cover _every_ field, however the mounts / port mappings are the finicky ones, will come back and add more test cases later.

Reply:
I suggest creating an issue with help-wanted for these things, the same way you did the shellcheck meta issue in k/k

Reply:
also, some fields like `ipFamily` are covered by validation, not perfect, but at least the bad config is not going through